server/store/string_store.go

```diff
@@ -9,6 +9,9 @@ import "fmt"
 // PrefixInfohash is the prefix to be used for infohashes.
 const PrefixInfohash = "ih-"
 
+// PrefixPeerID is the prefix to be used for client peer IDs.
+const PrefixPeerID = "pi-"
```

Comment:
TBH, I still don't like having any of these prefixes in the string store package. I think it should be the job of the middleware to make sure their prefixes don't collide with anyone else's.